Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: catalogs/unity_catalog/README.md

## Step 3. Configure the object store credentials

> Configure the credentials needed to connect to the Delta Lake tables provided by the Unity Catalog Connector, based on which object store they are located in.

Reviewer: ```suggestion
Configure credentials for the underlying Delta Lake tables object store.
```

### AWS S3

```yaml
params:
  unity_catalog_aws_access_key_id: ${env:AWS_ACCESS_KEY_ID}
  unity_catalog_aws_secret_access_key: ${env:AWS_SECRET_ACCESS_KEY}

>   unity_catalog_aws_region: <region> # i.e. us-east-1, us-west-2

Reviewer: ```suggestion
  unity_catalog_aws_region: <region> # E.g. us-east-1, us-west-2
```

>   unity_catalog_aws_region: <region>

Reviewer: Examples in comments.
